README.rst

```diff
@@ -59,6 +59,33 @@ There is one special method which allows transfer file between hosts.
         h2, "/path/to/file/on/h2/or/target/dir",
     )
 
+You can also mount devices.
+
+.. code:: python
+
+    from rrmngmnt import Host, RootUser
+    from rrmngmnt.filesystem import MountPoint
+
+    h = Host('10.11.12.13')
+    h.users.append(RootUser('123456'))
+    with h.fs.mount_point(
+      '//example.com/share', opts='ro,guest',
+      fstype='cifs', target='/mnt/netdisk'
+    ) as mp:
+      mp.fs.listdir(mp.target)
+
+    # OR
+
+    mp = MountPoint(
+      h.fs, 'example.com:/share', opts='ro,sync', fs_type='nfs'
+    )
+    mp.mount()
+    mp.fs.listdir(mp.target) # list mounted directory
```

Comment:
I would show `h.fs.listdir(mp.target)` instead of `mp.fs` from my point of view ...